worker/tests/testcase/entity/filter.rs

```diff
@@ -0,0 +1,38 @@
+use reearth_flow_action::{ActionValue, Port};
+
+use crate::helper::init_test_runner;
+
+#[tokio::test]
+async fn test_run() {
+    let executor = init_test_runner("entity/filter", vec!["filter.csv"]).await;
```

Comment:
>> [!NOTE]  
Make sure that init_test_runner is set to test_id and the list of files in test data as arguments.